docker-compose.local.yml

```diff
@@ -0,0 +1,8 @@
+version: "3"
+services:
+  nomis-prisoner-deletion-service:
+    build:
+      context: .
+    container_name: nomis-prisoner-deletion-service
+    deploy:
+      replicas: 0
```

Comment:
This overrides the base docker-compose file to stop running the nomis-prisoner-deletion service